packages/ui-components/src/Input/TextTypeInput.tsx

```diff
@@ -57,18 +52,16 @@ export const BaseTextInput: React.FC<InternalTextProps> = props => {
     type = "text",
     prefixElement,
     ariaLabelledBy,
-    existingPassword = false,
-    forgotPasswordLinkElement,
     ...rest
   } = props;
 
   const inputProps = {
+    id: id,
     className: classNames("crl-input", className, {
       "crl-input--prefix": prefixElement,
       "crl-input--suffix": suffix,
       invalid: error || invalid,
     }),
-    name: name,
```

Comment:
We don't destructure name as part of the props, so we can pass it as part of the rest on line 70.